ui/src/core/components/Button/Rounded/__tests__/ButtonIconRounded.spec.tsx

```diff
@@ -14,25 +14,72 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import { render, fireEvent, wait } from 'unit-test/testUtils';
-import ButtonIconRounded from '..';
+import React from "react";
+import { render, fireEvent, wait } from "unit-test/testUtils";
+import ButtonIconRounded from "..";
 
-test('render ButtonIconRounded default component', async () => {
-	const click = jest.fn();
-	const props = {
-		name: 'add',
-		children: 'button'
-	};
-	const { getByTestId } = render(
-		<ButtonIconRounded onClick={click} name={props.name}>
-			{props.children}
-		</ButtonIconRounded>
-	);
+test("render ButtonIconRounded default component", async () => {
+  const click = jest.fn();
+  const props = {
+    name: "add",
+    icon: "add",
+    children: "button"
+  };
+  const { getByTestId } = render(
+    <ButtonIconRounded onClick={click} name={props.name} icon={props.name}>
+      {props.children}
+    </ButtonIconRounded>
+  );
+  const Button = getByTestId(`button-iconRounded-${props.name}`);
+  const IconAdd = getByTestId(`icon-${props.name}`);
+  expect(Button && IconAdd).toBeInTheDocument();
+  fireEvent.click(Button);
+  wait(() => expect(click).toBeCalled());
+});
 
-	const Button = getByTestId(`button-iconRounded-${props.name}`);
+test("render ButtonIconRounded default component without default props", async () => {
+  const click = jest.fn();
+  const props = {
+    name: "add",
+    icon: "add",
+    children: "button"
+  };
+  const { getByTestId } = render(
+    <ButtonIconRounded
+      onClick={click}
+      name={props.name}
+      icon={props.name}
+      size="small"
+      backgroundColor="primary"
+    >
+      {props.children}
+    </ButtonIconRounded>
+  );
+  const Button = getByTestId(`button-iconRounded-${props.name}`);
+  const IconAdd = getByTestId(`icon-${props.name}`);
+  expect(Button && IconAdd).toBeInTheDocument();
```

Comment:
By chance, are you trying to do that ?

```suggestion
  expect(Button).toBeInTheDocument();
  expect(IconAdd).toBeInTheDocument();
```